eng/pipelines/runtime-richnav.yml

```diff
@@ -0,0 +1,40 @@
+trigger:
+  branches:
+    include:
+    - master
+    - dev/infrastructure
+    - release/*.*
+
+pr: none
+
+variables:
+- template: /eng/pipelines/common/variables.yml
+# TODO: (Consolidation) Switch away from old signing/validation variables from former Core-Setup. https://github.com/dotnet/runtime/issues/1027
+- name: TeamName
+  value: dotnet-core-acquisition
+# Set the target blob feed for package publish during official and validation builds.
+- name: _DotNetArtifactsCategory
+  value: .NETCore
+- name: _DotNetValidationArtifactsCategory
+  value: .NETCoreValidation
+
+stages:
+- stage: Build
+  jobs:
+  #
+  # Checkout repository
+  #
+  - template: /eng/pipelines/common/checkout-job.yml
+
+  #
+  # Build CoreCLR for Windows
+  #
+  - template: /eng/pipelines/common/xplat-setup.yml
+    parameters:
+      jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
```

Comment:
Why are we only building coreclr? Most of the C# code is built in libraries subset.

Reply:
Thanks so much for the question! Apologies for my naivety, I was confused about which builds should be included. Is there any other build I should be including? Also I mostly picked the architecture at random (we need to run on Windows) - does this look like an appropriate choice to you?